documentation/userguide.md

```diff
@@ -154,7 +154,7 @@ The profiler has several configuration options that can either be set as environ
 | :-------------------------------- | :--------------------------------------- | :--------------------------------------- |
 | COR_PROFILER_CONFIG               | Path                                     | Path to the profiler configuration file, e.g. `C:\Program Files\Coverage Profiler\profiler.yml` |
 | COR_PROFILER_TARGETDIR            | Path, default `c:/users/public/`         | Target directory for the trace files, e.g. `C:\Users\Public\Traces` |
-| COR_PROFILER_LIGHT_MODE           | `1` or `0`, default `0`                  | Enable ultra-light mode by disabling re-jitting of assemblies. |
+| COR_PROFILER_LIGHT_MODE           | `1` or `0`, default `1`                  | Enable ultra-light mode by disabling re-jitting of assemblies. |
```

Comment:
please amend the description with "Light mode must be disabled if you use the Native Image Cache"